components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -1642,6 +1642,12 @@ public class SQLQueries {
     public static final String GET_SHARED_APP_ID = "SELECT SHARED_APP_ID FROM SP_SHARED_APP WHERE " +
             "OWNER_ORG_ID = ? AND MAIN_APP_ID = ? AND SHARED_ORG_ID = ? ";
 
+    public static final String RETRIEVE_TOKEN_BINDING_BY_REFRESH_TOKEN =
+            "SELECT BINDING.TOKEN_BINDING_TYPE,BINDING.TOKEN_BINDING_VALUE,BINDING.TOKEN_BINDING_REF " +
+                    "FROM IDN_OAUTH2_ACCESS_TOKEN TOKEN LEFT JOIN IDN_OAUTH2_TOKEN_BINDING BINDING ON " +
+                    "TOKEN.TOKEN_ID=BINDING.TOKEN_ID WHERE TOKEN.REFRESH_TOKEN = ? " +
+                    "AND BINDING.TOKEN_BINDING_TYPE = ?";
```

Comment:
Lets format this query the same way others are

Reply:
Fixed in 3da41018b6a5b63ad5435d77db06c498340e3af0